.expeditor/release.omnibus.yml

```diff
@@ -25,7 +25,7 @@ builder-to-testers-map:
   mac_os_x-10.13-x86_64:
```

Comment:
You'll also want to switch your builder from 10.13 to 10.14. We deprecated 10.13 awhile ago.